### ISSUE TITLE
Can't find Barrel class.

### ISSUE THREAD

**Issue by helgihg** [jamesmontemagno/monkey-cache#49]
I'm hoping that this is the correct place to ask.

I've just installed MonkeyCache with NuGet on Visual Studio for Mac. The code examples in the README file show the use of a class called Barrel, such as Barrel.Current.Get and Barrel.Current.Add. Visual Studio does not find this class in my installation. It finds the interface IBarrel and BarrelUtils when MonkeyCache has been imported. I've tried messing around with both of those to try and achieve the same result as the code examples give but to no avail.

Code-completion in Visual Studio shows me 5 suggestions if I just type "MonkeyCache." and they are:

* BarrelUtils
* HttpCache
* HttpCacheExtensions
* HttpCacheRequestException
* IBarrel

The first four are classes, the last one is of course an interface.

Kinda stuck at this point. Any help appreciated.

**Comment by helgihg** [comment]
Of course I figured it out immediately after having asked. I needed to also install one of the storage mechanisms through NuGet.

This does not seem obvious to me by reading the README file, though. I'm tempted to send in a pull request with an updated README file, but also reluctant because I have yet to gain any experience whatsoever with the library. :) If I don't forget and no one else beats me to it, I'll try to remember to send in such a pull request once I've gotten the hang of using it.
